net/iputils/Makefile

```diff
@@ -0,0 +1,150 @@
+#
+# Copyright (C) 2006-2010 OpenWrt.org
+# Copyright (C) 2019 Noah Meyerhans <frodo@morgul.net>
+#
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=iputils
+PKG_VERSION:=20190709
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Noah Meyerhans <frodo@morgul.net>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE
+PKG_CPE_ID:=cpe:/a:iputils_project:iputils
+
+PKG_SOURCE:=iputils-s$(PKG_VERSION).tar.gz
```

Comment:
Ehm no.

Note that ? overides the filename, making it whatever you want.

I prefer that this be PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz for consistency.

Reply:
Fixed, thanks.